demos/handwritten_table_text_recognition_demo/python/README.md

```diff
@@ -0,0 +1,104 @@
+# Handwritten Chinese Table OCR Demo
+## Description
+This demo demonstrates using OpenCV and OpenVINO toolkit to recognize handwritten Chinese/Japanese tables and generate docx files with a GUI.
+
+
+## How It Works
+The demo uses the following model in the Intermediate Representation (IR) format:
+   * handwritten-japanese-recognition-0001
+   * handwritten-simplified-chinese-recognition-0001
+
+
+This demo workflow mainly has two parts:
+
+- The first part is the detection of the table. This part uses OpenCV to perform perspective transformation, and then detects the cells of the table.
+
+- The second part uses the cell images detected in the first part for handwritten text recognition with OpenVINO. It first reads a cell image and performs the preprocessing such as further binarization, resize and padding, and then the inference will start. After decoding the returned indexes into characters, the demo will display the predicted text. Optionally, you can generate the predicted text into a docx file.
+
+## Installation and dependencies
+### OS Requirement
+Ubuntu 18.04 or above.
+
+### OpenVINO Requirement
+
+You should install OpenVINO Toolkit (version >= 2020.3).
+
+For more OpenVINO toolkit installation information, please refer to [here](https://docs.openvinotoolkit.org/latest/openvino_docs_install_guides_installing_openvino_linux.html).
+
+### Other Dependencies
+This project requires Python 3.7.
```

Comment:
openvino has minimal requirements for python is 3.6
Do you use some python3.7 specific things in the demo?
Could you please relax dependencies on python to give more openvino users run your demo?

Reply:
Yes. I've tested this demo in pyton3.6. It worked. This requirement can be relaxed to python3.6.